server/swagger.json

```diff
@@ -5030,27 +5136,90 @@
         }
       }
     },
+    "SeverityColor": {
+      "type": "object",
+      "description":
+        "Color defines an encoding of a data value into color space",
+      "properties": {
+        "id": {
+          "description": "ID is the unique id of the cell color",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Type is how the color is used",
+          "type": "string",
+          "enum": ["emergency", "alert", "critical", "error", "warning", "notice", "info", "debug"]
+        },
+        "hex": {
+          "description": "Hex is the hex number of the color",
+          "type": "string",
+          "maxLength": 7,
+          "minLength": 7
+        },
+        "name": {
+          "description": "Name is the user-facing name of the hex color",
+          "type": "string"
+        }
+      }
+    },
     "RenamableField": {
       "description":
-        "renamableField describes a field that can be renamed and made visible or invisible",
+        "Describes a field that can be renamed and made visible or invisible",
```

Comment:
Would it be better to have a less general purpose type that is something like: LogColumn
```
{
  "tag": "tag-value", // influx 2 data storage meaning of tag, i.e. everything is a tag
  "display-text": "the displayed text"
  "widget": "dot-text",
  "pos": 0,
  "color": "some hex value"
} 
```

Reply:
that could be useful. `RenamableField` is a model already being used for table graph columns, so it seemed like something we could reuse here